scripts/zones/Bibiki_Bay/mobs/Peerifool.lua

```diff
@@ -19,7 +19,7 @@ entity.onMobDeath = function(mob, player, isKiller)
     local party = player:getParty()
     if #party > 1 then
         for _, v in ipairs(party) do
-            if player:getZone() == v:getZone() and v:getQuestStatus(2, 92) == QUEST_ACCEPTED then
+            if player:getZone() == v:getZone() and v:getQuestStatus(xi.quest.log_id.WINDURST, xi.quest.id.windurst.ONE_GOOD_DEED) == QUEST_ACCEPTED then
```

Comment:
This whole block can be used in the quest global file for onMobDeath, then you would only have to define what stage of the quest they are on